package.json

```diff
@@ -41,6 +41,7 @@
   },
   "scripts": {
     "test": "mocha",
+    "debugtest": "mocha --debug-brk $npm_package_options_mocha",
```

Comment:
Looks like it was just a temporary job that may have been left in by mistake. If it's intended then maybe add the script to the docs.